bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/communication/RotelConnector.java

```diff
@@ -458,6 +464,15 @@ public void sendCommand(RotelCommand cmd, @Nullable Integer value) throws RotelE
                                     messageStr += String.format("-%02d", -value);
                                 }
                                 break;
+                            case BALANCE_SET:
+                                if (value == 0) {
+                                    messageStr += "000";
+                                } else if (value > 0) {
+                                    messageStr += String.format("r%02d", value);
```

Comment:
Reading the specification, the r should be uppercase.

Reply:
you are right: in the specification, the balance command uses capital letters.
unfortunately, this does not work. 
at least with my device A14 MKII the following can be observed:

1) the spec requires upper case letters (e.g.: "balance_L05!").
2) it only works with lower case letters (e.g.: "balance_l05!")
3) the answer of the device uses upper case letters again (e.g.: "balance=L05$")
4) the display also shows capital letters without leading zero (e.g.: "BALANCE L5")

I attach two screenshots with directly observable communication in bash.
the world now offers several possibilities to adjust our behaviour:

1) my device is "wrong"; we need at least a second device to disprove this....
2) the spec is "wrong"...
3) ...

but you will understand that I do not want to change the lower case at this time...?

![a14 00 balance-spec](https://user-images.githubusercontent.com/10866978/158956411-23db8889-e5a7-48e9-8d7a-0a85e05da460.png)
![a14 01 command](https://user-images.githubusercontent.com/10866978/158956415-4e4e43fb-3f4a-410f-88ac-d7092eb26f79.png)
![a14 02 answer](https://user-images.githubusercontent.com/10866978/158956416-bd8619f1-456a-4747-b41f-4b3318a4bb72.png)
)

Reply:
2 options: either you think it is a general error in specifications (for all models) and in this case we keep your code but you add a comment in the code to explain the fact. Or you think it could only a bug for models A1x and in this case we have to cover the normal case and the specific A1x case.
WDYT ?

Reply:
Fix included in my PR